tests/ha/run.sh

```diff
@@ -19,8 +19,10 @@ function run() {
 	run_dm_master $WORK_DIR/master2 $MASTER_PORT2 $cur/conf/dm-master2.toml
 	check_rpc_alive $cur/../bin/check_master_online 127.0.0.1:$MASTER_PORT1
 	check_rpc_alive $cur/../bin/check_master_online 127.0.0.1:$MASTER_PORT2
-	check_metric $MASTER_PORT1 'start_leader_counter' 3 0 2
-	check_metric $MASTER_PORT2 'start_leader_counter' 3 -1 1 # master2 is not leader
+
+	# master1 or master2 join campaign
```

Comment:
could it be 
check_metric $MASTER_PORT1 'start_leader_counter' 3 0 2 || check_metric $MASTER_PORT1 'start_leader_counter' 3 0 2

Reply:
`3 -1 2` is too weak IMO

Reply:
but the metrics can be  0 or 1 ...